db/migrate/20170719210832_move_versions_table.rb

```diff
@@ -0,0 +1,5 @@
+class MoveVersionsTable < ActiveRecord::Migration
+  def change
+    rename_table :versions, :vestal_versions
```

Comment:
Can you confirm that the oracle adapter support this? This is the kind of thing where I'd be afraid it doesn't.